lib/bozo/preparers/file_templating.rb

```diff
@@ -143,10 +154,16 @@ def add_default_config_files(coordinator)
     # Adds the templates to the templating coordinator.
     def add_templates(coordinator)
       @template_globs.each do |glob|
-        coordinator.template_files glob
+        Dir[glob].each do |file|
+          coordinator.template_file file unless excluded_file?(file)
+        end
       end
     end
 
+    def excluded_file?(file)
+      @exclude_globs.any? { |glob| Dir[glob].include? (file) }
```

Comment:
Can we cache these? at the moment it'd hit the filesystem each time which might be a little slow